frontend/cypress/integration/dataSources.spec.js

```diff
@@ -56,4 +56,86 @@ describe('The Data Sources Page', () => {
         });
       });
   });
+
+  describe('data source update', () => {
+    it('freezes and deletes a data source', () => {
```

Comment:
I'd rather you split them into separate tests ... that way, it's very clear which one is broken.

You'll also need to add a 3rd test for the info button in the even that the freezing fails ... refer to screenshots below. Feel free to intercept this one.

![image](https://user-images.githubusercontent.com/5672438/117250189-87db2b80-ae4b-11eb-89b7-18983bef6bd7.png)

On clicking info, shows a modal:
![image](https://user-images.githubusercontent.com/5672438/117250286-a93c1780-ae4b-11eb-98b9-81f9cf9c25a6.png)

Reply:
The screenshots are for a datasets test, but the behaviour is the same for data sources. You'll naturally need to implement this for both.